### PR TITLE
Delete unnecessary links

### PR DESCRIPTION
**Pull request by JaeGeunBang**
### Description

Delete unnecessary links ([.idea/...](.idea/runConfigurations)) and change backtick (`./idea/...`) because the format of ./idea/... in the teamcity.md file is backtick.

<hr>

This PR has:
- [x] been self-reviewed.

### CODE REVIEW

File: dev/intellij-setup.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ setups of Druid.
 ## Project SDK
 
 The "SDK", configured for the Druid project, *must* be called `1.8`, to avoid annoying changes of
-[.idea/misc.xml](`.idea/misc.xml`) file, that you shouldn't check into your commits. If you don't have an SDK on your
+`.idea/misc.xml` file, that you shouldn't check into your commits. If you don't have an SDK on your
```
Comment (clintropolis) [inline dev/intellij-setup.md]: These still seem potentially useful to have as links, but they are currently broken, so how about changing to 
```
[`.idea/misc.xml`](/.idea/misc.xml)
```

File: dev/intellij-setup.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ This also assumes you have [ZooKeeper](http://zookeeper.apache.org/releases.html
 Before running the apps, you should do a `mvn clean install -DskipTests` in the Druid source in order to make sure directories are populated correctly.
 
 ## XML App Def
-You can configure application definitions in XML for import into IntelliJ. Below are a few examples. These should be placed in an XML file in [.idea/runConfigurations](.idea/runConfigurations) in the Druid source code.
+You can configure application definitions in XML for import into IntelliJ. Below are a few examples. These should be placed in an XML file in `.idea/runConfigurations` in the Druid source code.
```
Comment (clintropolis) [inline dev/intellij-setup.md]: Same suggestion about retaining links 
```
[`.idea/runConfigurations`](/.idea/runConfigurations)
```

Comment (JaeGeunBang) [reply dev/intellij-setup.md]: @clintropolis 
Thank you!

If the suggestion above, how about changing the ./idea/ in `teamcity.md` file?
https://github.com/apache/druid/blame/master/dev/teamcity.md#L87

```diff
- If you did everything right, file `.idea/inspectionProfiles/Druid.xml` should be updated with a new
+ If you did everything right, file [`.idea/inspectionProfiles/Druid.xml`](/.idea/inspectionProfiles/Druid.xml) should be updated with a new
```

Comment (clintropolis) [reply dev/intellij-setup.md]: sounds good :+1:

**Pull request closed without merging**